user-interface/src/main/java/life/qbic/datamanager/views/projects/project/measurements/MeasurementDetailsComponent.java

```diff
@@ -349,11 +343,20 @@ public void addListener(ComponentEventListener<MeasurementSelectionChangedEvent>
     listeners.add(listener);
   }
 
+  //TODO introduce custom tab with label and updateable count
```

Comment:
Is this still a TODO? How does this belong to your PR?

Reply:
not at all :/

Reply:
might come from @Steffengreiner s contribution